package.json

```diff
@@ -8,7 +8,8 @@
     "js:production": "npm run js -- -m",
     "css": "myth source/css/app.css build/app.css",
     "img": "imagemin-newer source/img/ build",
-    "html": "cp source/*.* build/",
+    "html": "static-site -s source/ -b build/",
+    "posthtml": "cp build/200/index.html build/200.html",
```

Comment:
yeah, put `prettyUrl: false` in your front matter

https://github.com/paulcpederson/static-site#front-matter